student_explorer/settings.py

```diff
@@ -353,7 +354,23 @@ def show_debug_toolbar(request):
 
 if DEBUG:
     from debug_toolbar import settings as dt_settings
-    DEBUG_TOOLBAR_PANELS = dt_settings.PANELS_DEFAULTS
+    DEBUG_TOOLBAR_PANELS = dt_settings.PANELS_DEFAULTS + ['cachalot.panels.CachalotPanel',]
     DEBUG_TOOLBAR_CONFIG = {"SHOW_TOOLBAR_CALLBACK" : show_debug_toolbar,}
     INSTALLED_APPS += ('debug_toolbar',)
-    MIDDLEWARE_CLASSES.append('debug_toolbar.middleware.DebugToolbarMiddleware')
\ No newline at end of file
+    MIDDLEWARE_CLASSES.append('debug_toolbar.middleware.DebugToolbarMiddleware')
+
+# Cache time to live
+CACHALOT_TIMEOUT = config("CACHE_TTL", default=60 * 15, cast=int)
+
+CACHE_OPTIONS = json.loads(config('CACHE_OPTIONS', default='{"CLIENT_CLASS": "django_redis.client.DefaultClient"}'))
+
+# Configure a cache
+CACHES = {
+    "default": {
+        "BACKEND": config('CACHE_BACKEND', default="django_redis.cache.RedisCache"),
```

Comment:
all those attribute defaults are defined in .env.sample, do we need to repeat the defaults here?

Reply:
I usually like to put the defaults in the code incase someone doesn't put the values in the env file and it will still work. I think the "better" default for this might actually just be 

**django.core.cache.backends.dummy.DummyCache**

Then it should still startup (without Redis) but just not cache anything. Let me try that one out instead. Based on the lack of forks this tool isn't getting any usage outside of Michigan but still should be runnable without optimal configuration.